credentials/development/gen_commissioner_dut_test_plan_table.py

```diff
@@ -86,8 +86,7 @@ def main():
         with open(path, 'r') as f:
             j = json.loads(f.read())
             success_expected = j['is_success_case'].lower() == 'true'
-            pid = 177 if 'fallback_encoding' in p else 32768
-            desc = TestInfo(desc=j['description'], dir=p, pid=pid)
+            desc = TestInfo(desc=j['description'], dir=p, pid=int(j['basic_info_pid']))
```

Comment:
What if it's not there? Suggest having a pre-check for presence in `j` with a log in case it's missing